rdkit/Chem/Draw/aggCanvas.py

```diff
@@ -20,8 +20,7 @@
 
 
 def convertColor(color):
-  color = (int(color[0] * 255), int(color[1] * 255), int(color[2] * 255))
-  return color
+  return int(color[0] * 255), int(color[1] * 255), int(color[2] * 255)
```

Comment:
Add the round brackets to make it clearer that a tuple is returned.

Reply:
I don't agree with it. 
The main reason came from the input as we already defined it as RGB color with three inputs.
Secondly, this function will unlikely to be changed in the future. Looking from the code, the input is the normalized color which have been divided by 255 previously. Python simply understand this type of return is always a tuple regardless of output

Reply:
I'm not talking about Python, I'm talking about someone looking at the code. People are reading code.

Reply:
OK, I got your idea! I would fix it in the next patch. Thanks you for your review